### PR TITLE
Add mitsobox

### PR DESCRIPTION
**Pull request by mrclemds**
Add mitsobox

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/mitsobox/tslint.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        "space-before-function-paren": false,
+        "space-within-parens": false,
+        "strict-export-declare-modifiers": false,
+        "trim-file": false,
+        "triple-equals": false,
+        "typedef-whitespace": false,
+        "unified-signatures": false,
+        "void-return": false,
+        "whitespace": false
+    }
```
Comment (andrewbranch) [inline types/mitsobox/tslint.json]: image

**Review by andrewbranch — CHANGES_REQUESTED** [review]

File: types/mitsobox/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+// Type definitions for mitsobox 0.1
+// Project: https://github.com/jimfilippou/mitsobox
+// Definitions by: MrClemds <https://github.com/MrClemds>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.7
```
Comment (andrewbranch) [inline types/mitsobox/index.d.ts]: Why does this require TypeScript 3.7?

**Review by andrewbranch — CHANGES_REQUESTED** [review]

File: types/mitsobox/index.d.ts (hunk 2 above)
Comment (mrclemds) [reply types/mitsobox/index.d.ts]: I thought it was necessary to indicate this information and so I added the one I use.

File: types/mitsobox/tslint.json (hunk 1 above)
Comment (mrclemds) [reply types/mitsobox/tslint.json]: Okay, I'm sorry, this is a mistake.

**Review by andrewbranch — APPROVED** [review]
Thanks for the updates!

**Pull request merged**